backend/compact-connect/stacks/api_lambda_stack/__init__.py

```diff
@@ -29,6 +30,17 @@ def __init__(
             **kwargs,
         )
 
+        # we only pass the API_BASE_URL env var if the API_DOMAIN_NAME is set
+        # this is because the API_BASE_URL is used by the feature flag client to call the flag check endpoint
```

Comment:
So, do the feature flags only work when we have a domain name defined, then? Sandboxes with no domain name will just use `fail_default`?